### PR TITLE
test: removed Schema from CommonModel test

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR removes any references to Schema from CommonModel test

### CODE REVIEW

File: test/models/CommonModel.spec.ts [hunk 1]
```diff
@@ -100,24 +99,24 @@ describe('CommonModel', () => {
 
   describe('additionalItems', () => {
     test('should return a Schema object', () => {
-      const doc: Schema = { additionalItems: { type: 'string' } };
-      const d = Schema.toSchema(doc) as Schema;
+      const doc: any = { additionalItems: { type: 'string' } };
```
Comment (magicmatatjahu) [inline test/models/CommonModel.spec.ts:102]: `any`... rejected! :trollface:

**Review by magicmatatjahu — DISMISSED** [review]
LGTM!

**Comment by jonaslagoni** [comment]
@magicmatatjahu sorry, had to force a new commit for Coverall to pass apparently 🧐

**Review by magicmatatjahu — APPROVED** [review]
LGTM!

**Comment by jonaslagoni** [comment]
Thanks 🙇

**Pull request merged**